rotors_gazebo_plugins/msgs/CommandMotorSpeed.proto

```diff
@@ -0,0 +1,6 @@
+syntax = "proto2";
+package gz_mav_msgs;
+
+message CommandMotorSpeed {
+  repeated float motor_speed = 1 [packed=true];
```

Comment:
Why not `float`? Are motor speed commands always `integers`? Or are you asking why is it not higher precision (i.e. `double`)?